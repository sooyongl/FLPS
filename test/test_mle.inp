data: file is test_mle.csv;

variable: names are Y Z X1 X2 item1-item20;
          usevariables are Y Z X1 X2 item1-item20;
          categorical = item1-item20;
          missing = all(-99);

analysis:

 TYPE = RANDOM;

model:

   F1 by item1-item20@1;

   [item1$1-item20$1];

   F1 on X1, X2;
   Y on Z X1 X2 F1;

   F1Z | F1 xwith Z;

   Y on F1Z;