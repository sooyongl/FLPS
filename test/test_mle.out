Mplus VERSION 8.6
MUTHEN & MUTHEN
03/05/2022   9:54 AM

INPUT INSTRUCTIONS

  data: file is test_mle.csv;

  variable: names are Y Z X1 X2 item1-item20;
            usevariables are Y Z X1 X2 item1-item20;
            categorical = item1-item20;
            missing = all(-99);

  analysis:

   TYPE = RANDOM;

  model:

     F1 by item1-item20@1;

     [item1$1-item20$1];

     F1 on X1 X2;
     Y on Z X1 X2 F1;

     F1Z | F1 xwith Z;

     Y on F1Z;



INPUT READING TERMINATED NORMALLY




SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         500

Number of dependent variables                                   21
Number of independent variables                                  3
Number of continuous latent variables                            2

Observed dependent variables

  Continuous
   Y

  Binary and ordered categorical (ordinal)
   ITEM1       ITEM2       ITEM3       ITEM4       ITEM5       ITEM6
   ITEM7       ITEM8       ITEM9       ITEM10      ITEM11      ITEM12
   ITEM13      ITEM14      ITEM15      ITEM16      ITEM17      ITEM18
   ITEM19      ITEM20

Observed independent variables
   Z           X1          X2

Continuous latent variables
   F1          F1Z


Estimator                                                      MLR
Information matrix                                        OBSERVED
Optimization Specifications for the Quasi-Newton Algorithm for
Continuous Outcomes
  Maximum number of iterations                                 100
  Convergence criterion                                  0.100D-05
Optimization Specifications for the EM Algorithm
  Maximum number of iterations                                 500
  Convergence criteria
    Loglikelihood change                                 0.100D-02
    Relative loglikelihood change                        0.100D-05
    Derivative                                           0.100D-02
Optimization Specifications for the M step of the EM Algorithm for
Categorical Latent variables
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
Optimization Specifications for the M step of the EM Algorithm for
Censored, Binary or Ordered Categorical (Ordinal), Unordered
Categorical (Nominal) and Count Outcomes
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
  Maximum value for logit thresholds                            15
  Minimum value for logit thresholds                           -15
  Minimum expected cell size for chi-square              0.100D-01
Maximum number of iterations for H1                           2000
Convergence criterion for H1                             0.100D-03
Optimization algorithm                                         EMA
Integration Specifications
  Type                                                    STANDARD
  Number of integration points                                  15
  Dimensions of numerical integration                            1
  Adaptive quadrature                                           ON
Link                                                         LOGIT
Cholesky                                                       OFF

Input data file(s)
  test_mle.csv
Input data format  FREE


SUMMARY OF DATA

     Number of missing data patterns             2
     Number of y missing data patterns           1
     Number of u missing data patterns           2


COVARIANCE COVERAGE OF DATA

Minimum covariance coverage value   0.100


     PROPORTION OF DATA PRESENT


           Covariance Coverage
              ITEM1         ITEM2         ITEM3         ITEM4         ITEM5
              ________      ________      ________      ________      ________
 ITEM1          0.500
 ITEM2          0.500         0.500
 ITEM3          0.500         0.500         0.500
 ITEM4          0.500         0.500         0.500         0.500
 ITEM5          0.500         0.500         0.500         0.500         0.500
 ITEM6          0.500         0.500         0.500         0.500         0.500
 ITEM7          0.500         0.500         0.500         0.500         0.500
 ITEM8          0.500         0.500         0.500         0.500         0.500
 ITEM9          0.500         0.500         0.500         0.500         0.500
 ITEM10         0.500         0.500         0.500         0.500         0.500
 ITEM11         0.500         0.500         0.500         0.500         0.500
 ITEM12         0.500         0.500         0.500         0.500         0.500
 ITEM13         0.500         0.500         0.500         0.500         0.500
 ITEM14         0.500         0.500         0.500         0.500         0.500
 ITEM15         0.500         0.500         0.500         0.500         0.500
 ITEM16         0.500         0.500         0.500         0.500         0.500
 ITEM17         0.500         0.500         0.500         0.500         0.500
 ITEM18         0.500         0.500         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500
 Y              0.500         0.500         0.500         0.500         0.500
 Z              0.500         0.500         0.500         0.500         0.500
 X1             0.500         0.500         0.500         0.500         0.500
 X2             0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              ITEM6         ITEM7         ITEM8         ITEM9         ITEM10
              ________      ________      ________      ________      ________
 ITEM6          0.500
 ITEM7          0.500         0.500
 ITEM8          0.500         0.500         0.500
 ITEM9          0.500         0.500         0.500         0.500
 ITEM10         0.500         0.500         0.500         0.500         0.500
 ITEM11         0.500         0.500         0.500         0.500         0.500
 ITEM12         0.500         0.500         0.500         0.500         0.500
 ITEM13         0.500         0.500         0.500         0.500         0.500
 ITEM14         0.500         0.500         0.500         0.500         0.500
 ITEM15         0.500         0.500         0.500         0.500         0.500
 ITEM16         0.500         0.500         0.500         0.500         0.500
 ITEM17         0.500         0.500         0.500         0.500         0.500
 ITEM18         0.500         0.500         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500
 Y              0.500         0.500         0.500         0.500         0.500
 Z              0.500         0.500         0.500         0.500         0.500
 X1             0.500         0.500         0.500         0.500         0.500
 X2             0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              ITEM11        ITEM12        ITEM13        ITEM14        ITEM15
              ________      ________      ________      ________      ________
 ITEM11         0.500
 ITEM12         0.500         0.500
 ITEM13         0.500         0.500         0.500
 ITEM14         0.500         0.500         0.500         0.500
 ITEM15         0.500         0.500         0.500         0.500         0.500
 ITEM16         0.500         0.500         0.500         0.500         0.500
 ITEM17         0.500         0.500         0.500         0.500         0.500
 ITEM18         0.500         0.500         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500
 Y              0.500         0.500         0.500         0.500         0.500
 Z              0.500         0.500         0.500         0.500         0.500
 X1             0.500         0.500         0.500         0.500         0.500
 X2             0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              ITEM16        ITEM17        ITEM18        ITEM19        ITEM20
              ________      ________      ________      ________      ________
 ITEM16         0.500
 ITEM17         0.500         0.500
 ITEM18         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500
 Y              0.500         0.500         0.500         0.500         0.500
 Z              0.500         0.500         0.500         0.500         0.500
 X1             0.500         0.500         0.500         0.500         0.500
 X2             0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              Y             Z             X1            X2
              ________      ________      ________      ________
 Y              1.000
 Z              1.000         1.000
 X1             1.000         1.000         1.000
 X2             1.000         1.000         1.000         1.000


     PROPORTION OF DATA PRESENT FOR U


           Covariance Coverage
              ITEM1         ITEM2         ITEM3         ITEM4         ITEM5
              ________      ________      ________      ________      ________
 ITEM1          0.500
 ITEM2          0.500         0.500
 ITEM3          0.500         0.500         0.500
 ITEM4          0.500         0.500         0.500         0.500
 ITEM5          0.500         0.500         0.500         0.500         0.500
 ITEM6          0.500         0.500         0.500         0.500         0.500
 ITEM7          0.500         0.500         0.500         0.500         0.500
 ITEM8          0.500         0.500         0.500         0.500         0.500
 ITEM9          0.500         0.500         0.500         0.500         0.500
 ITEM10         0.500         0.500         0.500         0.500         0.500
 ITEM11         0.500         0.500         0.500         0.500         0.500
 ITEM12         0.500         0.500         0.500         0.500         0.500
 ITEM13         0.500         0.500         0.500         0.500         0.500
 ITEM14         0.500         0.500         0.500         0.500         0.500
 ITEM15         0.500         0.500         0.500         0.500         0.500
 ITEM16         0.500         0.500         0.500         0.500         0.500
 ITEM17         0.500         0.500         0.500         0.500         0.500
 ITEM18         0.500         0.500         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              ITEM6         ITEM7         ITEM8         ITEM9         ITEM10
              ________      ________      ________      ________      ________
 ITEM6          0.500
 ITEM7          0.500         0.500
 ITEM8          0.500         0.500         0.500
 ITEM9          0.500         0.500         0.500         0.500
 ITEM10         0.500         0.500         0.500         0.500         0.500
 ITEM11         0.500         0.500         0.500         0.500         0.500
 ITEM12         0.500         0.500         0.500         0.500         0.500
 ITEM13         0.500         0.500         0.500         0.500         0.500
 ITEM14         0.500         0.500         0.500         0.500         0.500
 ITEM15         0.500         0.500         0.500         0.500         0.500
 ITEM16         0.500         0.500         0.500         0.500         0.500
 ITEM17         0.500         0.500         0.500         0.500         0.500
 ITEM18         0.500         0.500         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              ITEM11        ITEM12        ITEM13        ITEM14        ITEM15
              ________      ________      ________      ________      ________
 ITEM11         0.500
 ITEM12         0.500         0.500
 ITEM13         0.500         0.500         0.500
 ITEM14         0.500         0.500         0.500         0.500
 ITEM15         0.500         0.500         0.500         0.500         0.500
 ITEM16         0.500         0.500         0.500         0.500         0.500
 ITEM17         0.500         0.500         0.500         0.500         0.500
 ITEM18         0.500         0.500         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500


           Covariance Coverage
              ITEM16        ITEM17        ITEM18        ITEM19        ITEM20
              ________      ________      ________      ________      ________
 ITEM16         0.500
 ITEM17         0.500         0.500
 ITEM18         0.500         0.500         0.500
 ITEM19         0.500         0.500         0.500         0.500
 ITEM20         0.500         0.500         0.500         0.500         0.500


     PROPORTION OF DATA PRESENT FOR Y


           Covariance Coverage
              Y             Z             X1            X2
              ________      ________      ________      ________
 Y              1.000
 Z              1.000         1.000
 X1             1.000         1.000         1.000
 X2             1.000         1.000         1.000         1.000


UNIVARIATE PROPORTIONS AND COUNTS FOR CATEGORICAL VARIABLES

    ITEM1
      Category 1    0.360           90.000
      Category 2    0.640          160.000
    ITEM2
      Category 1    0.400          100.000
      Category 2    0.600          150.000
    ITEM3
      Category 1    0.484          121.000
      Category 2    0.516          129.000
    ITEM4
      Category 1    0.092           23.000
      Category 2    0.908          227.000
    ITEM5
      Category 1    0.548          137.000
      Category 2    0.452          113.000
    ITEM6
      Category 1    0.144           36.000
      Category 2    0.856          214.000
    ITEM7
      Category 1    0.608          152.000
      Category 2    0.392           98.000
    ITEM8
      Category 1    0.668          167.000
      Category 2    0.332           83.000
    ITEM9
      Category 1    0.448          112.000
      Category 2    0.552          138.000
    ITEM10
      Category 1    0.284           71.000
      Category 2    0.716          179.000
    ITEM11
      Category 1    0.324           81.000
      Category 2    0.676          169.000
    ITEM12
      Category 1    0.440          110.000
      Category 2    0.560          140.000
    ITEM13
      Category 1    0.620          155.000
      Category 2    0.380           95.000
    ITEM14
      Category 1    0.784          196.000
      Category 2    0.216           54.000
    ITEM15
      Category 1    0.160           40.000
      Category 2    0.840          210.000
    ITEM16
      Category 1    0.404          101.000
      Category 2    0.596          149.000
    ITEM17
      Category 1    0.376           94.000
      Category 2    0.624          156.000
    ITEM18
      Category 1    0.632          158.000
      Category 2    0.368           92.000
    ITEM19
      Category 1    0.448          112.000
      Category 2    0.552          138.000
    ITEM20
      Category 1    0.396           99.000
      Category 2    0.604          151.000



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     Y                     0.356       0.082      -2.513    0.20%      -0.437      0.076      0.342
             500.000       0.874      -0.171       3.266    0.20%       0.583      1.108
     Z                     0.500       0.000       0.000   50.00%       0.000      0.000      0.500
             500.000       0.250      -2.000       1.000   50.00%       1.000      1.000
     X1                   -0.073      -0.253      -3.042    0.20%      -0.860     -0.298     -0.060
             500.000       0.907       0.031       2.609    0.20%       0.176      0.806
     X2                    0.518      -0.072       0.000   48.20%       0.000      0.000      1.000
             500.000       0.250      -1.995       1.000   51.80%       1.000      1.000


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       30

Loglikelihood

          H0 Value                       -2972.305
          H0 Scaling Correction Factor      0.9987
            for MLR

Information Criteria

          Akaike (AIC)                    6004.611
          Bayesian (BIC)                  6131.049
          Sample-Size Adjusted BIC        6035.827
            (n* = (n + 2) / 24)



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 F1       BY
    ITEM1              1.000      0.000    999.000    999.000
    ITEM2              1.000      0.000    999.000    999.000
    ITEM3              1.000      0.000    999.000    999.000
    ITEM4              1.000      0.000    999.000    999.000
    ITEM5              1.000      0.000    999.000    999.000
    ITEM6              1.000      0.000    999.000    999.000
    ITEM7              1.000      0.000    999.000    999.000
    ITEM8              1.000      0.000    999.000    999.000
    ITEM9              1.000      0.000    999.000    999.000
    ITEM10             1.000      0.000    999.000    999.000
    ITEM11             1.000      0.000    999.000    999.000
    ITEM12             1.000      0.000    999.000    999.000
    ITEM13             1.000      0.000    999.000    999.000
    ITEM14             1.000      0.000    999.000    999.000
    ITEM15             1.000      0.000    999.000    999.000
    ITEM16             1.000      0.000    999.000    999.000
    ITEM17             1.000      0.000    999.000    999.000
    ITEM18             1.000      0.000    999.000    999.000
    ITEM19             1.000      0.000    999.000    999.000
    ITEM20             1.000      0.000    999.000    999.000

 F1         ON
    X1                -0.908      0.073    -12.367      0.000
    X2                 0.419      0.140      2.997      0.003

 Y          ON
    F1                 0.300      0.041      7.374      0.000
    F1Z               -0.165      0.040     -4.144      0.000

 Y          ON
    Z                  0.214      0.044      4.816      0.000
    X1                 0.987      0.035     28.100      0.000
    X2                 0.506      0.046     10.920      0.000

 Intercepts
    Y                 -0.003      0.041     -0.072      0.943

 Thresholds
    ITEM1$1           -0.478      0.176     -2.716      0.007
    ITEM2$1           -0.257      0.175     -1.464      0.143
    ITEM3$1            0.198      0.182      1.090      0.276
    ITEM4$1           -2.642      0.254    -10.396      0.000
    ITEM5$1            0.542      0.178      3.040      0.002
    ITEM6$1           -2.029      0.219     -9.264      0.000
    ITEM7$1            0.865      0.183      4.725      0.000
    ITEM8$1            1.210      0.181      6.695      0.000
    ITEM9$1            0.008      0.170      0.047      0.962
    ITEM10$1          -0.941      0.189     -4.992      0.000
    ITEM11$1          -0.694      0.179     -3.886      0.000
    ITEM12$1          -0.035      0.170     -0.204      0.839
    ITEM13$1           0.932      0.181      5.138      0.000
    ITEM14$1           1.967      0.193     10.205      0.000
    ITEM15$1          -1.876      0.212     -8.832      0.000
    ITEM16$1          -0.232      0.182     -1.278      0.201
    ITEM17$1          -0.391      0.177     -2.213      0.027
    ITEM18$1           1.003      0.181      5.540      0.000
    ITEM19$1           0.006      0.182      0.031      0.976
    ITEM20$1          -0.281      0.185     -1.519      0.129

 Residual Variances
    Y                  0.176      0.016     11.161      0.000
    F1                 1.011      0.139      7.296      0.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.756E-05
       (ratio of smallest to largest eigenvalue)


DIAGRAM INFORMATION

  Use View Diagram under the Diagram menu in the Mplus Editor to view the diagram.
  If running Mplus from the Mplus Diagrammer, the diagram opens automatically.

  Diagram output
    c:\users\lee\box\git\flps\test\test_mle.dgm

     Beginning Time:  09:54:00
        Ending Time:  09:54:00
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2021 Muthen & Muthen
